.jscsrc

```diff
@@ -1,48 +0,0 @@
-{
-  "excludeFiles": ["src/ngLocale/**"],
```

Comment:
>
> I wonder why we excluded ngLocale/** and if we should keep doing it...
>
Because they're largely auto-generated AFAIK. Most of them are JSONs,
though, so disabling a couple of rules fixes that. Note that only JSCS was
disabled, not JSHint so disabling checking these files in ESLint would
decrease our linting scope.

-- 
Michał Gołębiowski